### ISSUE TITLE
Multiple syntax errors when running on Py3

### ISSUE THREAD

**Issue by nonhermitian** [pyscf/pyscf#189]
Installation highlights the following syntax errors:

```
File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/tools/localizer.py", line 164
    print self.gradient
             ^
SyntaxError: Missing parentheses in call to 'print'
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/pbc/tools/tril.py", line 14
    nints = sum([isinstance(x,int) for x in kp,kq,kr])
                                              ^
SyntaxError: invalid syntax
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/pbc/mpicc/kccsd_rhf.py", line 253
    s0,s1= [slice(min(x),max(x)+1) for x in ranges0,ranges1]
                                                   ^
SyntaxError: invalid syntax
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/pbc/lib/arnoldi.py", line 81
    print ""
           ^
SyntaxError: Missing parentheses in call to 'print'
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/pbc/lib/davidson.py", line 37
    print "NOT A MATRIX in SVD_CUT !",mat.shape
                                    ^
SyntaxError: Missing parentheses in call to 'print'
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/rt/tdscf.py", line 37
    print "matrixpower: Minimal Eigenvalue =", np.min(s)
                                            ^
SyntaxError: invalid syntax
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/cc/davidson.py", line 305
    print rdic['e']

SyntaxError: Missing parentheses in call to 'print'
```

```
 File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/lo/nlocal.py", line 37
    print numpy.linalg.norm(diff)
              ^
SyntaxError: invalid syntax
```

```
 File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/lo/pmloc.py", line 58
    print '[pm_loc_kernel]'
                          ^
SyntaxError: Missing parentheses in call to 'print'
```

```
  File "/Users/paul/psi4conda/lib/python3.6/site-packages/pyscf/lo/ulocal.py", line 31
    print '\nLowdin population for LMOs:'
                                        ^
SyntaxError: Missing parentheses in call to 'print'
```

**Comment by sunqm** [comment]
These errors are produced by the setup program.  The python3 incompatible module will not be installed.  The rest programs were implemented in the Python 3 compatible form and can be used in Python 3 environment without problems.

These erroneous modules have been removed in the dev branch which will be the next release (pyscf-1.5).  In the released pyscf-1.4.* versions, because some users are using the functions from these codes, I will keep them there as they are.

**Comment by nonhermitian** [comment]
Actually they are installed.  They just cannot be run due to the syntax errors.  For example:
```
In [1]: run /anaconda3/lib/python3.6/site-packages/pyscf/lo/nlocal.py
  File "/anaconda3/lib/python3.6/site-packages/pyscf/lo/nlocal.py", line 37
    print numpy.linalg.norm(diff)
              ^
SyntaxError: invalid syntax
```
